plugins/braintree-payment/src/providers/payment-braintree/src/core/braintree-base.ts

```diff
@@ -438,7 +442,14 @@ class BraintreeBase extends AbstractPaymentProvider<BraintreeOptions> {
           countryCodeAlpha2: this.sanitizeCountryCodeAlpha2(context.shipping_address?.country_code),
         }
       : undefined;
-
+      let cart: CartDTO | undefined;
+      if(context.items?.[0]?.cart_id) {
+        cart = await this.cartService.retrieveCart(context.items?.[0]?.cart_id as string);
+          if (!cart) {
+            throw new MedusaError(MedusaError.Types.NOT_FOUND, 'Cart not found');
+          }
+        }
```

Comment:
Hmmm @SGFGOV it might not have been the best choice to pull in the cart module here. The braintree payment plugin can be used for cart purchases, but also order edit, which doesn't have a cart. Is there a reason you needed to pull in the cart here, instead of including more information here: https://github.com/lambda-curry/360training/blob/dev/apps/medusa/src/workflows/complete-360-cart/complete-360-cart.ts#L254C13-L254C25